qiskit_ibm/ibm_backend_service.py

```diff
@@ -61,26 +62,36 @@ class IBMBackendService:
         job = provider.backend.job(<JOB_ID>)
     """
 
-    def __init__(self, provider: 'ibm_provider.IBMProvider') -> None:
+    def __init__(self, provider: 'ibm_provider.IBMProvider', hgp: HubGroupProject) -> None:
         """IBMBackendService constructor.
 
         Args:
             provider: IBM Quantum account provider.
+            hgp: default hub/group/project to use for the service.
         """
         super().__init__()
-
         self._provider = provider
+        self._hgp = hgp
```

Comment:
nit - can we rename this to `_default_hgp`? So it doesn't seem as if it's the _only_ hgp associated with this service.